manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
I tend to agree. Everything looks good to me then, no need to worry about my prior comment. If you are good with this PR I'll merge it.